Tribler/community/multichain/database.py

```diff
@@ -39,13 +39,21 @@
 
 CREATE TABLE option(key TEXT PRIMARY KEY, value BLOB);
 INSERT INTO option(key, value) VALUES('database_version', '""" + str(LATEST_DB_VERSION) + u"""');
+
+CREATE INDEX IF NOT EXISTS idx_mc_public_key ON multi_chain(public_key);
+CREATE INDEX IF NOT EXISTS idx_mc_link_public_key ON multi_chain(link_public_key);
 """
 
 upgrade_to_version_2_script = u"""
 DROP TABLE IF EXISTS multi_chain;
 DROP TABLE IF EXISTS option;
 """
 
+create_index_script = u"""
```

Comment:
Still, couldn't you declare the index_script before 43-44 and append the index_script to 42. This way, you get the same functionality without duplicating the code.